### ISSUE TITLE
Note to IP banned players

### ISSUE THREAD

**Issue by JoeyIsLit** [finndev/PokeBuddy#818]
Honestly people getting ip banned are just careless. I have used 3 separate accounts all level 23+ and no bans. Why? Because I wait for hours in between changing locations. It's all about patience, don't go blaming the bot because YOU'RE MANUALLY TELEPORTING ALL OVER. It's really annoying trying to find real issues when it's all about these ip bans non stop or update ideas. Just google it or something and once you find a solution, post it.

**Comment by LammonDT** [comment]
Not true! As my last post stated, I use the iOS 'tap to walk' app from iEnchantify and I tried manipulating a ban. Straight off I teleported to pier39 when logging in. Pier 39 in San Francisco. Pokestops worked. Continued to catch Pokemon for a bit. Worked fine. Turned the run all the way up, ran till the passenger verification came up and kept running, without exiting it. still wasn't enough to get me ip banned. I use multiple pc when botting. Literally anything this bot does I could do in the app. Everything except the always perfect throws. I think that's relevant.

**Comment by JoeyIsLit** [comment]
I have multiple accounts running at once and at times I forget to wait because it's late. I'll end up teleporting to the UK then New York then California. Zero bans on my end. It's all about BEING CAREFUL and not RECKLESS

**Comment by LammonDT** [comment]
The only time the game logs your location, to detect you for a soft ban, is when you spin a poke stop or catch a Pokemon. Things that register to your journal. I, at first, would run the bot and in fear of deleting Pokemon I wanted, would turn the bot off every hour on the hour and delete extra Pokemon. Never got a soft ban once for that. Ever.

**Comment by LammonDT** [comment]
The ip ban are completely separate issue. As, with a soft ban, from teleportin ect you get locked out of the game on your account. On this new ip thing once they detect you, the ip gets locked out. Not your account.

**Comment by DatFIXER** [comment]
Same, yesterday got IP banned while i don't even log with the phone or change coords... so this guy is just throwing bullshit.

**Comment by xXbrucyjucyXx** [comment]
yeah....use a ip address spoofer or an vpn.......if your ip is band switch to a different vpn.......its kinda simple

**Comment by JoeyIsLit** [comment]
It just really bugs me that people are spamming the thread with this "issue" when I really doubt it's the bots fault at all

**Comment by drdetroit1** [comment]
OP teleporting gets you a soft ban not ip ban maybe google it or something before blaming others ;)

**Comment by xXbrucyjucyXx** [comment]
there are like 4 threads about this issue its super irritating to see people doing dumb shit and then expecting there hands to get held through there idiocy.......read the threads addressing your issues we have been helld nice in pointing out specific programs even to people...

**Comment by DatFIXER** [comment]
I came out with this:
This worked for me, still working:
- If you've pointed the new Hatching Eggs feature, undone that
- Try to delete all folders to make a new fresh "setup"
- Run only one account
- Mov Speed 13km/ph

As long i don't use more than one account, is working fine.

**Comment by xXbrucyjucyXx** [comment]
jesus christ just use this.....http://download.vpngate.jp/common/cd.aspx/vpngate-client-2016.08.08-build-9613.136171.zip for fucks sake.....go as fast as you want

**Comment by LammonDT** [comment]
As more people continue to read your bullshit "fixes" then post a thread for themselves. Cool they haven't detected you immediately again. Won't last long bud. Couple hours here and there isn't the 14 hour runs could do before with no interference. It isn't the bot? Well niantic has updated a few times this last week. The Api has barely been finished that long. And as the guys who cracked the api said, there are still issues in the api that they didn't complete, for instance altitude issues sending indefinite flags blah blah.

**Comment by LammonDT** [comment]
Not throwing out bullshit, I just followed along the whole time on them cracking the Api so he could even make the bot work again. Was in their discord channel the whole time. Lol they laughed saying they don't promote botting nor care if people have issues with botting. As that wasn't their goal. Lol

**Comment by RoseCorpse** [comment]
well, as I said in #793 post, I'm not using my cellphone, just Nox and bot with the same coordinates, my bot is working like a charm.

**Comment by LammonDT** [comment]
And bruicy that is an implausible fix for some, as those with Comcast are disabled from changing their ip. That's why people report no fix for the Vpn. As they are seen through the mask.

**Comment by LammonDT** [comment]
With Comcast or xfinity, force send their static ip, no one with Comcast will experience a fix for longer than 5 min no matter the vpn. We have tried and tested dozens.

**Comment by K1llercat** [comment]
Seems like the "inventory menu" (that one from the button bottom right) is the issue. If I leave it closed, the bot runs perfect without any issues. Other than that, I left "Evolve" unchecked "running" at 10km/h and left Incubator unchecked too.

**Comment by DatFIXER** [comment]
My menu is everytime opened and doesn't cause any issue; seem that the incubators are the cause.

**Comment by r0nalxd** [comment]
For more info, check my post in issue #707
